lib/napa/cli/templates/new/Gemfile.tt

```diff
@@ -1,9 +1,9 @@
 source 'https://rubygems.org'
-ruby "2.0.0"
+ruby '<%= RUBY_VERSION %>'
```

Comment:
I think it's a way to specify a dependency on a version of Ruby with Bundler (http://bundler.io/gemfile_ruby.html).
And I know Heroku uses this to determine the ruby version to run your app (https://devcenter.heroku.com/articles/ruby-versions), but not sure about `rbenv`, though.

The `RUBY_VERSION` constant gives you the version of Ruby that you are currently running.

I think that if you put the ruby version dependency on the Gemfile, it makes more sense to use the ruby version on which you are executing `napa` instead of `2.0.0` (which might not even be installed on the system)